src/Symfony/Component/Messenger/Tests/Transport/Doctrine/ConnectionTest.php

```diff
@@ -209,7 +209,7 @@ public function buildConfigurationProvider(): iterable
             'expectedAutoSetup' => false,
         ];
 
-        yield 'options from options array wins over options from dsn' => [
+        yield 'options from dsn array wins over options from options' => [
```

Comment:
We should instead be referring to URI not DSN https://github.com/symfony/symfony/pull/36999#issuecomment-635865593

Reply:
maybe, but this is not the purpose of this PR. 
Dsn is consistent with the current code and documentation: https://symfony.com/doc/current/messenger.html#transports-async-queued-messages